src/facade/dragonfly_connection.cc

```diff
@@ -164,9 +164,13 @@ void OpenTrafficLogger(string_view base_path) {
 #else
   LOG(WARNING) << "Traffic logger is only supported on Linux";
 #endif
+
+  // Write version, incremental numbering :)
+  tl_traffic_logger.log_file->Write(io::Bytes{2});
```

Comment:
1. what's "io::Bytes{2}" ?
2. Can you pls handle the error code as well?

Reply:
(1) The version 😆

Reply:
I understand but I do not understand the code. Bytes should wrap a uint8_t buffer, and you pass a literal int. 
Can you please make it more explicit?